templates/init/functions/typescript/tslint.json

```diff
@@ -107,9 +107,6 @@
     // Warns if function overloads could be unified into a single function with optional or rest parameters.
     "unified-signatures": {"severity": "warning"},
 
-    // Warns if code has an import or variable that is unused.
```

Comment:
why are we taking this out?

Reply:
The rule is deprecated: https://github.com/palantir/tslint/pull/3919

Though there's talk of bringing it back: https://github.com/palantir/tslint/issues/4100